### ISSUE TITLE
`tempfile.SpooledTemporaryFile.encoding` is `None` when the `encoding` parameter is `utf-8`

### ISSUE THREAD

**Issue by jtrh** [fyntex/lib-cl-sii-python#120]
**Code**: https://github.com/fyntex/lib-cl-sii-python/blob/267c1a55423c7ed6ced718810bb4bd8164b65014/cl_sii/libs/io_utils.py#L61-L63

**Test**: https://github.com/fyntex/lib-cl-sii-python/blob/267c1a55423c7ed6ced718810bb4bd8164b65014/tests/test_libs_io_utils.py#L70-L73

This issue affects Python 3.7.2, but versions starting from 3.7.6 are not affected.

**Comment by jtrh** [comment]
@glarrain The cause of https://github.com/fyntex/lib-cl-sii-python/blob/267c1a55423c7ed6ced718810bb4bd8164b65014/cl_sii/libs/io_utils.py#L61-L63 is [bpo-26730](https://bugs.python.org/issue26730).

**Comment by jtrh** [comment]
Fixed in https://github.com/python/cpython/pull/17400.

**Comment by glarrain** [comment]
Wow! How did you figure this one out?

**Comment by jtrh** [comment]
> Wow! How did you figure this one out?

When testing support for Python 3.8 I noticed that the bug wasn't there, and I knew that the bug was present in 3.7.2, so I concluded that the bug was fixed somewhere between those versions. A review of the VCS history of file `tempfile.py`, focused on class `SpooledTemporaryFile`, did the rest.

> In which CPython versions (full) was it fixed?

The bug was fixed in Python 3.7.6, and is still fixed in 3.8.2.
